src/helpers.js

```diff
@@ -17,6 +17,11 @@ Marionette.isNodeAttached = function(el) {
   return Backbone.$.contains(document.documentElement, el);
 };
 
+// Merge `keys` from `options` onto `this`
+Marionette.mergeOptions = function(options, keys) {
+  if (!options) { return; }
+  _.extend(this, _.pick(options, keys));
```

Comment:
I can see a `abstractViewProperties: ['_isShown']` or whatever for properties that `Marionette.AbstractView` uses. Subsequent child classes wouldn't override it, yet it would provide some degree of documentation and introspection for the curious of heart (think inspector).